protocol/messenger_saved_address.go

```diff
@@ -32,6 +35,45 @@ func (m *Messenger) garbageCollectRemovedSavedAddresses() error {
 	return m.savedAddressesManager.DeleteSoftRemovedSavedAddresses(uint64(time.Now().AddDate(0, 0, -30).Unix()))
 }
 
+func (m *Messenger) setInstallationHostname() error {
```

Comment:
@Samyoul is this file a good place for installation hostname? Won't `messenger_identity.go` or `messenger_config.go` be more suitable?

P.S. I now realize this file's name should have been `messenger_saved_addresses.go`

Reply:
Thank you, that is a good point, I will move this function `messenger_identity.go`